view/frontend/templates/breadcheckout/grouped.phtml

```diff
@@ -188,11 +188,16 @@
                 breadConfig.actAsLabel = false;
             }
 
-            if(cartSizeFinancing.enabled){
-                var cartSizeFinancingId = cartSizeFinancing.id;
-                var cartSizeThreshold = cartSizeFinancing.threshold;
+            if(isFinancing.enabled && isFinancing.mode.cart){
+                var cartSizeFinancingId = financingInfo.id;
+                var cartSizeThreshold = financingInfo.threshold;
                 var itemsPriceSum = items.reduce(function(sum, item) { return sum + item.price * item.quantity }, 0) / 100;
                 breadConfig.financingProgramId = (itemsPriceSum >= cartSizeThreshold) ? cartSizeFinancingId : 'null';
+            } else if(financingInfo.enabled && financingInfo.mode.sku){
```

Comment:
why the difference in the code with how we check sku stuff here vs. the cartview.phtml? If we could standardize that, that'd be good I think, and this code here I think looks a bit more preferable, what do you think?

Reply:
On PDP we only need to check one sku, that is of main parent item, on cartview, minicart and checkout we need to check all items that are on quote